datadog/resource_datadog_dashboard.go

```diff
@@ -1174,6 +1188,22 @@ func getGroupDefinitionSchema() map[string]*schema.Schema {
 			Optional:    true,
 			Description: "The title of the group.",
 		},
+		"background_color": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "Background color of the group title.",
```

Comment:
What kinds of values are allowed for this field?

Reply:
The background_color field accepts any string, but only these values will render a colored header: `vivid_blue`, `vivid_purple`, `vivid_pink`, `vivid_orange`, `vivid_yellow`, `vivid_green`, `blue`, `purple`, `pink`, `orange`, `yellow`, `green`, `gray` or `white`.
-> I updated the description with the values.